provider/datagen/tests/test-baked-source.rs

```diff
@@ -0,0 +1,49 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+extern crate alloc;
+
+mod testutil {
+    include!("testutil.rs");
+}
+
+#[allow(unused_imports)]
+mod baked_data {
```

Comment:
suggestion: isolate this in the const block.

Reply:
done